dll/cpl/desk/background.c

```diff
@@ -69,6 +69,21 @@ typedef struct _BACKGROUND_DATA
 
 GLOBAL_DATA g_GlobalData;
 
+static BOOL
+IsBitmapFile(PCWSTR pszPath)
+{
+    BYTE bfh[sizeof(BITMAPFILEHEADER)];
+    BOOL success = FALSE;
+    HANDLE hFile = CreateFileW(pszPath, GENERIC_READ, FILE_SHARE_READ | FILE_SHARE_WRITE | FILE_SHARE_DELETE,
+                               NULL, OPEN_EXISTING, 0, NULL);
+    if (hFile != INVALID_HANDLE_VALUE)
+    {
+        DWORD read;
+        success = ReadFile(hFile, bfh, sizeof(bfh), &read, NULL) && read == sizeof(bfh);
+        CloseHandle(hFile);
+    }
+    return success && bfh[0] == 'B' && bfh[1] == 'M';
```

Comment:
or maybe check [`(bfh.bfType == 0x4d42); /* BM */`](https://git.reactos.org/?p=reactos.git&a=search&h=28b802b229e151568687593c7eeae84085de9022&st=grep&s=bfType) if bfh was of type `BITMAPFILEHEADER` ?